sanity/schemas/page.ts

```diff
@@ -43,6 +50,52 @@ const PageSchema = {
             },
           ],
         },
+        {
+          name: 'piechart-section',
+          title: 'Kakediagram',
+          type: 'object',
+          icon: Progress75Icon,
+          fields: [
```

Comment:
Should we have an attribute with the date the data was fetched and confirmed?

Reply:
Good idea! I'll add it